extensions/amp-access-fewcents/OWNERS

```diff
@@ -0,0 +1,10 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/main/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [{name: 'ampproject/wg-components'}],
```

Comment:
can you change this to wg-monetization? You should also add yourself.

Reply:
Done.